src/ParserConfigFile.cpp

```
@@ -17,6 +18,7 @@
 Raytracer::ParserConfigFile::ParserConfigFile(
     const std::string &filename, const std::vector<std::string> &plugins)
     : _plugins(plugins) {
+  _currentFilePath = filename;
```

Cppcheck (2.10-2)

[performance] Variable '_currentFilePath' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)